github/data_source_github_release.go

```diff
@@ -147,7 +151,8 @@ func dataSourceGithubReleaseRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("published_at", release.GetPublishedAt())
 	d.Set("url", release.GetURL())
 	d.Set("html_url", release.GetHTMLURL())
-	d.Set("asserts_url", release.GetAssetsURL())
+	d.Set("asserts_url", release.GetAssertsURL())
```

Comment:
hi @smiller171! I'd recommend leaving this as `d.Set("asserts_url", release.GetAssetsURL())` so existing users/configs can still access this value. If we go in the direction of Deprecating the `asserts_url` attribute, within the "assets_url" block, after L87, you can add the key/value pair: `Deprecated: "Use `assets_url` instead",`. these docs provide additional examples: https://www.terraform.io/docs/extend/best-practices/deprecations.html#renaming-a-computed-attribute

Reply:
Thanks for the link. I hope to have time to fix this soon